proposed/phpdoc-meta.md

```diff
@@ -46,16 +44,17 @@ Cons:
 ### 3.1 Goals
 
 * Provide a complete technical definition, or schema, of the PHPDoc notation.
-* Introduce new concepts matching best practices or design patterns in use today and in the foreseeable future.
-* Deprecate old concepts that are replaced by newer concepts or are no longer in use in today's PHP landscape.
+* Introduce new concepts matching best practices or design patterns in use today.
 
 ### 3.2 Non-Goals
 
-* This PSR does not provide a recommendation on how and when to use the concepts described in this document,
-  so it is not a coding standard.
+* This PSR does not introduce new concepts that guess at matching best practices or design patterns in the foreseeable
+  future.
+* This PSR does not provide a recommendation on how and when to use the concepts described in this document, so it is
+  not a coding standard.
 * This PSR facilitates the creation of annotations by allowing the notation needed for Symfony/Doctrine style
```

Comment:
I'd remove this as a non-goal, but not add it as a goal so we have some intentional ambiguity there honestly.